downstream/modules/platform/ref-controller-variables.adoc

```diff
@@ -8,7 +8,10 @@
 | *`admin_password`* | The password for an administration user to access the UI when the installation is complete.
 
 Passwords must be enclosed in quotes when they are provided in plain text in the inventory file.
-| *`automation_controller_main_url`* | For an alternative front end URL needed for SSO configuration, provide the URL.
+
+| *`automation_controller_main_url`* | The full routeable URL used by EDA to connect to a controller host. This URL is required if there is no Automation Controller configured in inventory.
```

Comment:
'Automation Controller' -> 'automation controller'. I did a quick scan of the usage throughout the guide and sentence case is used in most instances.

Reply:
in this particular instance,  the noted content is a variable and does not use sentence case.